docs/Hooks.md

```diff
@@ -339,11 +339,22 @@ Note: using an arrow function will break the binding of this to the Fastify inst
 
 <a name="route-hooks"></a>
 ## Route level hooks
-You can declare one or more custom `preValidation` and `preHandler` hook(s) that will be unique for the route.
+You can declare one or more custom `onRequest`, `preParsing`, `preValidation`, `preHandler` and `preSerialization` hook(s) that will be unique for the route.
```

Comment:
The implementation looks to support array values for these route hooks but the documentation does not mention such a feature.

Reply:
@jsumners PTAL.